lib/graphql/mutations.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from "@apollo/client"
+
```

Comment:
For graphql queries we should be using ```@galoymoney/client``` repo

Reply:
Okay. How do I access that?